lasagne/layers/merge.py

```diff
@@ -304,9 +302,8 @@ class ElemwiseMergeLayer(MergeLayer):
 
     Parameters
     ----------
-    incomings : a list of :class:`Layer` instances or tuples
-        the layers feeding into this layer, or expected input shapes,
-        with all incoming shapes being equal
+    incomings : Unless `cropping` is given, all shapes must be equal, except
+        for dimensions that are undefined (``None``) or broadcastable (``1``).
```

Comment:
We still need the previous text, it tells what types are accepted. You can add your part in the end, after `being equal`.